client/src/components/Login.jsx

```diff
@@ -1,16 +1,68 @@
-import React from 'react';
-import { Link } from 'react-router-dom';
+import React, { useState, useContext } from 'react';
+import { Link, useNavigate } from 'react-router-dom';
 import Button from '@mui/material/Button';
 import CssBaseline from '@mui/material/CssBaseline';
 import TextField from '@mui/material/TextField';
 import Grid from '@mui/material/Grid';
 import Box from '@mui/material/Box';
 import Typography from '@mui/material/Typography';
 import Container from '@mui/material/Container';
+import Alert from '@material-ui/core/Alert';
+import { AuthContext } from '../contexts/index.jsx';
 
 function Login() {
-  const handleSubmit = (event) => {
+  const navigate = useNavigate();
+  // State
+  const [email, setEmail] = useState();
+  const [password, setPassword] = useState();
+  const [error, setError] = useState('');
+  const [isLoading, setLoading] = useState(false);
+
+  // Context
+  const { login } = useContext(AuthContext);
+
+  // Methods
+  const onChange = (e) => {
+    const { id, value } = e.target;
+    if (id === 'email') setEmail(value);
+    if (id === 'password') setPassword(value);
+  };
+
+  const handleSubmit = async (event) => {
     event.preventDefault();
```

Comment:
on line 25 the event was referenced with 'e' and on line 31 its referenced with 'event'. I think we should try to pick one format and stick with it

Reply:
Good point. I'll update that